river/optim/schedulers.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, loss: optim.losses.Loss, alpha=1e-4):
         self.alpha = alpha
 
         typw = math.sqrt(1.0 / math.sqrt(self.alpha))
-        initial_eta0 = typw / max(1.0, self.loss.gradient(-typw, 1.0))
+        initial_eta0 = typw / max(1.0, self.loss.gradient(True, -typw))
```

Comment:
I changed this because it wasn't correct. In the past I had copy/pasted this from scikit-learn, but the order isn't the same.

Reply:
Cool! I didn't know bool variables were cast automatically in arithmetic operations. Did some tests and it works like a charm